test/wallet-claim-test.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * wallet-claim-test.js - Test Name Claim Updates for hsd
+ * Copyright (c) 2020, Mark Tyneway (MIT License).
+ * https://github.com/handshake-org/hsd
+ */
+
+'use strict';
+
+const assert = require('bsert');
+const FullNode = require('../lib/node/fullnode');
+const common = require('./util/common');
+const ownership = require('../lib/covenants/ownership');
+const Address = require('../lib/primitives/address');
+
+const ports = {
+  p2p: 14331,
+  node: 14332,
+  wallet: 14333
+};
+
+const node = new FullNode({
+  memory: true,
+  network: 'regtest',
+  plugins: [require('../lib/wallet/plugin')],
+  workers: false,
+  env: {
+    'HSD_WALLET_HTTP_PORT': ports.wallet.toString()
+  }
+});
+
+const walletdb = node.require('walletdb');
+let wallet, claimPeriod;
+
+describe('Wallet Claims', function() {
+  before(async () => {
+    // Allow for fake ownership claims to be mined.
+    ownership.ignore = true;
```

Comment:
The other "fakeclaim" tests put `ownership.ignore = false` in the `finally` block of a try/catch. I think that's best practice because there are failure states where `after` doesn't get executed (i.e. if `before` fails) and could affect other tests.